be/src/runtime/routine_load/data_consumer.cpp

```diff
@@ -203,11 +204,14 @@ Status KafkaDataConsumer::group_consume(TimedBlockingQueue<RdKafka::Message*>* q
             }
             ++received_rows;
             break;
-        case RdKafka::ERR__TIMED_OUT:
+        case RdKafka::ERR__TIMED_OUT: {
             // leave the status as OK, because this may happened
             // if there is no data in kafka.
-            LOG(INFO) << "kafka consume timeout: " << _id;
+            std::stringstream ss;
+            ss << msg->errstr() << ", partition " << msg->partition() << " offset " << msg->offset();
```

Comment:
Is the partition/offset of msg accurate? Since the consumer may be assigned with multi partitions. Which partition would be returned here ?

Reply:
this msg's partition/offset and not consumer, but i'm not sure is it accurate if request timeout.